stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/PhoneNumberFormatter.kt

```diff
@@ -408,7 +408,6 @@ internal sealed class PhoneNumberFormatter {
             "HT" to Metadata(prefix = "+509", regionCode = "HT", pattern = "## ## ####"),
             "PE" to Metadata(prefix = "+51", regionCode = "PE", pattern = "### ### ###"),
             "MX" to Metadata(prefix = "+52", regionCode = "MX", pattern = "### ### ####"),
-            "CY" to Metadata(prefix = "+537", regionCode = "CY"),
```

Comment:
See the existing line that correctly handles this here: https://github.com/stripe/stripe-android/blob/cd2ba357c2bbd41e7e6e08a3505e5d8261a460ae/stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/PhoneNumberFormatter.kt#L357